### ISSUE TITLE
`terraform_required_version`: false positive with multiple `terraform` blocks

### ISSUE THREAD

**Issue by bendrucker** [terraform-linters/tflint#1451]
## Introduction

When multiple `terraform` blocks are defined, TFLint is unable to detect that `required_version` has been set, resulting in a warning for valid configuration. This is a regression in 0.39.0. 

## Expected Behavior

No reported issues.

## Actual behavior

```console
1 issue(s) found:

Warning: terraform "required_version" attribute is required (terraform_required_version)

  on  line 0:
   (source code not available)
```

## Steps to Reproduce

```hcl
terraform {
  required_version = ">= 1, < 2"
}

terraform {
  cloud {
    organization = "my-org"

    workspaces {
      name = "my-ws"
    }
  }
}
```

```sh
tflint --enable-rule terraform_required_version
```

## Additional Context

```console
$ tflint -v
TFLint version 0.39.0
$ terraform -v
Terraform v1.2.5
```

**Comment by yermulnik** [comment]
@bendrucker Any ETA on when the fix is going to be bundled into release? Thanks.
